src/Runner.Common/JobServerQueue.cs

```diff
@@ -204,6 +230,29 @@ public void QueueFileUpload(Guid timelineId, Guid timelineRecordId, string type,
             _fileUploadQueue.Enqueue(newFile);
         }
 
+        public void QueueResultsSummaryUpload(Guid timelineId, Guid timelineRecordId, string type, string name, string path, string stepId, bool deleteSource)
```

Comment:
do we need to pass `Guid timelineId, Guid timelineRecordId` anymore?

Reply:
Yeah we still need `planId`, `jobId` and `stepId` in Results.